reference/channels.v3.yml

```diff
@@ -502,11 +502,118 @@ paths:
+                        "id": 1,
+                        "url": "https://kittens.mybigcommerce.com/",
+                        "channel_id": 1,
+                        "created_at": "2018-01-04T04:15:50.000Z",
+                        "updated_at": "2018-01-04T04:15:50.000Z",
+                        "urls": [
+                            {
+                                "url": "https://kittens.mybigcommerce.com",
+                                "type": "primary",
+                                "created_at": "2022-03-22T13:08:59Z",
+                                "updated_at": "2022-03-22T13:08:59Z"
+                            },
+                            {
+                                "url": "https://store-abc.store.bcdev",
+                                "type": "canonical",
+                                "created_at": "2022-03-22T13:08:59Z",
+                                "updated_at": "2022-03-22T13:08:59Z"
+                            },
+                            {
+                                "url": "https://checkout.kittens.mybigcommerce.com",
+                                "type": "checkout",
+                                "created_at": "2022-03-23T10:21:11Z",
+                                "updated_at": "2022-03-29T13:44:22Z"
+                            }
+                        ],
+                        "is_checkout_url_customized": true
+                    }
+        '422':
+          description: Unprocessable Entity. Your checkout and storefront must be within the same main domain like `example.com` and `subdomain.example.com`
+          content:
+            application/json:
+              schema:
+                type: object
+                x-examples:
+                  example-1:
+                    status: 422
+                    title: 'Incorrect checkout url https://bc.kittykat.me. Your checkout and storefront must be within the same main domain like “main.com” and “subdomain.main.com”'
+                    type: 'https://developer.bigcommerce.com/api-docs/getting-started/api-status-codes'
+                    errors: {}
+                properties:
+                  status:
+                    type: integer
+                    default: 422
+                  title:
+                    type: string
+                    example: 'Incorrect checkout url https://checkout.example.com. Your checkout and storefront must be within the same main domain like “main.com” and “subdomain.main.com'
+                  type:
+                    type: string
+                    default: 'https://developer.bigcommerce.com/api-docs/getting-started/api-status-codes'
+                  errors:
+                    type: object
+    delete:
+      summary: Delete a Site's Checkout URL
+      operationId: delete-checkout-url
+      description: 'Deletes a site''s checkout URL. After deletion, a shared checkout URL is used.'
+      tags:
+        - Channel Site Checkout Url
+      responses:
```

Comment:
does `delete` have any error responses?

Reply:
I wasn't able to produce one. Putting in a channel that doesn't exist still returns a `200` with empty `data` and `meta` objects.